### DISCUSSION TITLE
How to integrate Giscus to your Astro Blog

### DISCUSSION THREAD

**Discussion by giscus** [maxpou/maxpou.fr#200] [General]
# How to integrate Giscus to your Astro Blog

Learn how to integrate Giscus (GitHub-powered comments), into your Astro blog. How to deal with dark mode and styles customization.

http://localhost:4321/blog/2024/2024-08-23-giscus-astro/

**Comment by maxpou** [comment]
![](https://media3.giphy.com/media/v1.Y2lkPTc5MGI3NjExeWR4ODlkbWhjbGo3bThwcXZleHhyNWQxa3lkaDh0M3JmaGpyM2o5ZiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/IThjAlJnD9WNO/200.webp)

**Comment by bzsgbq** [comment]
This is so helpful to me!

**Comment by athena-sh** [comment]
Helpful, thanks!

**Comment by michaelangeloio** [comment]
thank you!
